tests/www/apply/test_process.py

```diff
@@ -1383,6 +1383,133 @@ def test_refuse_step_bypass(self, client):
         # Trying to access step 2 without providing data for step 1 redirects to step 1
         assertRedirects(response, refusal_reason_url)
 
+    def test_add_to_pool_from_prescriber(self, client, snapshot, mailoutbox):
+        initial_state = random.choice(
+            [
+                job_applications_enums.JobApplicationState.NEW,
+                job_applications_enums.JobApplicationState.PROCESSING,
+                job_applications_enums.JobApplicationState.POSTPONED,
+                job_applications_enums.JobApplicationState.CANCELLED,
+                job_applications_enums.JobApplicationState.OBSOLETE,
+            ]
+        )
```

Comment:
```suggestion
        initial_state = random.choice(JobApplicationWorkflow.CAN_BE_ADDED_TO_POOL_STATES)
```

+ idem les deux autres ?